libraries/AP_InertialSensor/AP_InertialSensor_L3G4200D.h

```diff
@@ -1,7 +1,8 @@
-#pragma once
+
+#ifndef __AP_INERTIAL_SENSOR_L3G4200D2_H__
+#define __AP_INERTIAL_SENSOR_L3G4200D2_H__
 
 #include <AP_HAL/AP_HAL.h>
-#if CONFIG_HAL_BOARD == HAL_BOARD_LINUX
```

Comment:
please keep this as we definitly don't want this driver on other board for now

Reply:
done